lib/conekta-button.scss

```diff
@@ -5,238 +5,239 @@ $font-family: "Montserrat", "Helvetica Neue", "Helvetica", Arial, sans-serif;
 @import url(http://fonts.googleapis.com/css?family=Montserrat);
 
 .#{$modal_name} * { 
-	box-sizing: border-box;
+    box-sizing: border-box;
 }
 
 .#{$modal_name}{
-	display: block;
-	font-size: 62.5%; 
-	
-	overflow-x: auto;
-	overflow-y: scroll;
-	position: fixed;
-	top: 0px;
-	right: 0px;
-	bottom: 0px;
-	left: 0px;
-	z-index: 1050;
-
-	padding-right: 25px;
-	padding-left: 25px;
-	outline: 0px none;
-	opacity: 1;
-	background: rgba(0,0,0,0.4);
-	font-family: $font-family;
-
-	.#{$modal_name}-dialog{
-		position: relative;
-		width: 413px;
-		margin: 5% auto;
-		margin-top: 104px;
-		background: #FFFFFF;
-		border-radius: 4px;
-		box-shadow: 0px 5px 15px rgba(0, 0, 0, 0.5);
-
-		.#{$modal_name}-content{
-
-			.#{$modal_name}-header{
-				margin: 0 26px;
-				padding-top: 22px;
-
-				h1{
-					margin: 0;
-					font-size: 27px;
-					letter-spacing: .01em;
-
-				}
-			}
-
-			.#{$modal_name}-body{
-				margin: 0 26px;
-			}
-
-			.#{$modal_name}-footer{
-				padding: 20px 26px 13px;
-				border-bottom-right-radius: 4px;
-				border-bottom-left-radius: 4px;
-				
-				background-color: #F9FAFB;
-				border-top: 1px solid #EEEEEE;
-
-
-				button{
-					font-family: $font-family;
-
-					display: inline-block;
-					transition: all 0.15s ease-out 0s;
-					border-radius: 6px;
-					color: white;
-					font-weight: bold;
-					letter-spacing: 1px;
-					margin-bottom: 1em;
-					padding: 1em;
-					text-align: center;
-					border: none;
-					box-shadow: none;
-					text-transform: uppercase;
-
-					&#paynow{
-						background-color: #6D48CC;
-						&:hover{
-							background-color: darken(#6D48CC, 10%);
-						}
-						float: right;
-					}
-
-					&#cancel{
-						background-color: #859093;
-						&:hover{
-							background-color: darken(#859093, 10%);
-						}
-					}
-				}
-			}
-
-
-
-
-
-		}
-	}
+    display: block;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background, bottom, display, font-family, font-size, left, opacity, outline, overflow-x, overflow-y, padding-left, padding-right, position, right, top, z-index